pylint/checkers/spelling.py

```diff
@@ -44,6 +48,27 @@
 table = maketrans("", "")
 
 
+class WordsWithDigigtsFilter(Filter):
+    """
+    Skips words with digits
+    """
+    _pattern = re.compile(r"\d")
+
+    def _skip(self, word):
+        if self._pattern.findall(word):
```

Comment:
maybe `any(char.isdigit() for char in word)`? No need to use regular expression for such a simple case